src/TabBar.js

```diff
@@ -70,7 +71,7 @@ export default class TabBar<T: Route<*>>
     renderLabel: PropTypes.func,
     renderIndicator: PropTypes.func,
     onTabPress: PropTypes.func,
-    tabStyle: View.propTypes.style,
+    tabStyle: ViewPropTypes.style,
```

Comment:
This is an unfortunate breaking change. I think we might need to remove this prop type entirely for now :(